README.md

```diff
@@ -13,7 +13,20 @@
    </p>
 </div>
 
-### Warning! This repo/software is under active development
+<div align="center">
+  <p>
+    <b>
+       Warning! This repo/software is under active development
+    </b>
+   </p>
+</div>
+
+## Important links
+
+**[Install Skandha](https://etherspot.fyi/skandha/installation)**
+| [Chains supported](https://etherspot.fyi/skandha/chains)
+| [Bundler Fee history](https://etherspot.fyi/skandha/feehistory)
+| [Prime SDK docs](https://etherspot.fyi/prime-sdk/intro)
```

Comment:
Do we need a reference to Prime here? I would take this out. We can maybe add this under miscellaneous section.

Reply:
ya sure, I'll remove it